Remotion/Web/Development.WebTesting/WebDriver/Factories/Chrome/ChromeBrowserFactory.cs

```diff
@@ -55,7 +55,7 @@ public IBrowserSession CreateBrowser (DriverConfiguration configuration)
 
       var extendedChromeOptions = _chromeConfiguration.CreateChromeOptions();
       if (configuration.Headless)
-        extendedChromeOptions.AddArgument("headless");
+        extendedChromeOptions.AddArgument("headless=new");
```

Comment:
Do you want to keep this as a distinct commit? if so, please update the commit messages.